src/components/Wrappers/withClaimReturned.js

```diff
@@ -15,6 +66,180 @@ const withClaimReturned = WrappedComponent => class withClaimReturnedComponent e
     };
   }
 
+  refundTransfers = async () => {
+    const getAccounts = (itemBarcode, userId) => {
+      const {
+        mutator: {
+          accounts: {
+            GET,
+          },
+        }
+      } = this.props;
+      const lostStatus = 'Lost item fee';
+      const processingStatus = 'Lost item processing fee';
```

Comment:
Instead of hard-coding strings like this, maybe make a list of possible values in `src/constants.js`? Then you can put all the potential fee statuses together, all the processing statuses together, all the payment statuses (see below on line 93) etc.

Reply:
These strings were changed to the file constants.js